src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewItemAccessibleObject.cs

```
@@ -0,0 +1,327 @@
+                }
+            }
+
+            /// <summary>
+            ///  Gets the accessible role.
+            /// </summary>
+            public override AccessibleRole Role => AccessibleRole.ListItem;
+
+            /// <summary>
+            ///  Gets the accessible state.
+            /// </summary>
+            public override AccessibleStates State
+            {
+                get
+                {
+                    AccessibleStates state = AccessibleStates.Selectable | AccessibleStates.Focusable | AccessibleStates.MultiSelectable;
+
+                    if (_owningListView.SelectedItems.Contains(_owningItem))
+                    {
+                        return state |= AccessibleStates.Selected | AccessibleStates.Focused;
+                    }
+
+                    if (_systemIAccessible != null)
+                    {
+                        return state |= (AccessibleStates)(_systemIAccessible.get_accState(GetChildId()));
+                    }
+
+                    return state;
+                }
+            }
+
+            internal override void AddToSelection() => SelectItem();
+
+            public override string DefaultAction => SR.AccessibleActionDoubleClick;
+
+            public override void DoDefaultAction() => SetFocus();
+
+            internal override UiaCore.IRawElementProviderFragment? FragmentNavigate(UiaCore.NavigateDirection direction)
+            {
+                switch (direction)
+                {
+                    case UiaCore.NavigateDirection.Parent:
+                        return _owningGroup != null
+                            ? _owningGroup.AccessibilityObject
+                            : _owningListView.AccessibilityObject;
+                    case UiaCore.NavigateDirection.NextSibling:
+                        if (_owningGroup != null)
+                        {
+                            var owningGroupAccessibleObject = _owningGroup.AccessibilityObject as ListViewGroupAccessibleObject;
```

Can `_owningGroup.AccessibilityObject` be anything but `ListViewGroupAccessibleObject`?
If not, then 
```suggestion
                            var owningGroupAccessibleObject = (ListViewGroupAccessibleObject)_owningGroup.AccessibilityObject;
```

This way if there is incorrect type is used we'll find out immediately, instead of hiding the error.

ditto below

---

fixed